pkg/component/shared/istio_test.go

```diff
@@ -279,7 +282,7 @@ var _ = Describe("Istio", func() {
 				labels,
 				&externalTrafficPolicy,
 				serviceExternalIP,
-				zone)).To(MatchError("at least one ingress gateway must be present before adding further ones"))
+				zone, false)).To(MatchError("at least one ingress gateway must be present before adding further ones"))
```

Comment:
Please add a line break after the `,` so that it fits to the rest.

Reply:
Done.